Tests/Remora.Discord.Tests/Samples/Objects/USER/USER.optionals.json

```diff
@@ -2,5 +2,6 @@
   "id": "999999999999999999",
   "username": "none",
   "discriminator": "9999",
+  "global_name": "none",
```

Comment:
This (along with the rest of the sample files) need to be updated in order to comply with the spec. This variant should not have `global_name` in the payload.

All other sample files (i.e, the ones that don't define the property in their type) can have the property removed as well, since the object used when in a nested property should be the minimal valid structure (i.e, the one in XX.optionals.json). 

You should still retain them in UserMention and the gateway events, though, since they directly define/use the property.